Model/CreditCardPaymentMethod.php

```diff
@@ -0,0 +1,192 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Model;
+
+use OxidEsales\Eshop\Application\Model\Payment;
+use OxidEsales\Eshop\Core\Config;
+use OxidEsales\Eshop\Core\Registry;
+use Wirecard\Oxid\Core\Helper;
+
+use Wirecard\PaymentSdk\Config\Config as Wirecard_Config;
```

Comment:
Please rename according to the standard, i suggest PaymentSdkConfig

Reply:
👍